tests/types/consistency.test.ts

```diff
@@ -1,14 +1,15 @@
 import { assert } from "chai";
 import fs from "fs";
-import * as types from "../../src/types";
+import * as sszTypes from "../../src/types";
```

Comment:
Technically it's still should be types? because all types are being imported?

Reply:
We're using this variable to extract ssz types. The only things its looping through (at run-time) are the ssz type variables.

But I mainly changed this variable name b/c I wanted to name another variable `types` (see L12) which corresponds to the typescript type declarations.

This PR works by now also storing every typescript type declaration, so we can figure out things like Shard == uint64.